version/VERSION

```diff
@@ -1 +1 @@
-1.16.3
\ No newline at end of file
+1.16.1
```

Comment:
```suggestion
1.16.0
```

@sgmiller can you actually reset this to 1.16.0 since we'll need to publish that again?

Reply:
Yeah, good point